crates/router/src/connector/aci/transformers.rs

```diff
@@ -91,9 +91,37 @@ impl TryFrom<&api_models::payments::WalletData> for PaymentDetails {
                     account_id: None,
                 }))
             }
-            _ => Err(errors::ConnectorError::NotImplemented(
-                "Payment method".to_string(),
-            ))?,
+            api_models::payments::WalletData::MbWayRedirect(_)
+            | api_models::payments::WalletData::AliPayRedirect(_)
```

Comment:
Same for `AlipayRedirect`

Reply:
Can you have a look now? I have made the requested changes.

Reply:
Can you have a look now? I have made the requested changes.

Reply:
Can you have a look now? I have made the requested changes.

Reply:
Done